src/gluonts/mx/model/tft/_network.py

```diff
@@ -151,7 +151,8 @@ def __init__(
         d_var: int,
         d_hidden: int,
         n_head: int,
-        n_output: int,
+        #n_output: int,
```

Comment:
Could this be removed instead of commented?

Reply:
Yes, it sneaked into it :)